fast_carpenter/summary/binning_config.py

```diff
@@ -61,7 +61,7 @@ def bin_one_dimension(low=None, high=None, nbins=None, edges=None,
     # - bins: {nbins: 6 , low: 1  , high: 5 , overflow: True}
     # - bins: {edges: [0, 200., 900], overflow: True}
     if all([x is not None for x in (nbins, low, high)]):
-        bin_obj = np.linspace(low, high, nbins + 1)
+        bin_obj = np.linspace(eval(str(low)), eval(str(high)), eval(str(nbins)) + 1)
```

Comment:
I really like the functionality this gives, but I'm a little nervous to use `eval` like this here since I think it's a security risk in this sort of case because it runs any string passed in.  You can do some things to make it less insecure, but it would be nice to find a better "maths calculator" since we're just looking at simple arithmetic.  It's probably overkill, but numexpr might actually be able to help, and we can just provide empty dicts for the local and global arguments of numexpr.evaluate so it can only handle constants.   How does that sound?

Reply:
Ah yeah, good point! I'll have a look at using numexpr!